src/config/enhance-octokit.ts

```diff
@@ -29,11 +29,11 @@ const instrumentRequests = (octokit: GitHubAPI) => {
 			throw new RateLimitingError(rateLimitReset);
 		}
 
-		if (error.status === 403) {
-			// delaying for an hour
-			logger.warn({ err: error }, "GitHub returned 403");
-			throw new RateLimitingError(new Date().getTime() / 1000 + 60 * 60);
```

Comment:
previously, we would throw a `RateLimitingError` on every 403 from GitHub, which is wrong